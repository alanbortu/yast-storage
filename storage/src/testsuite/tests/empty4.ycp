// testedfiles: helper1b.ycp
{
    include "./lib/setup-system.ycp";

    setup_system("empty");

    include "./lib/helper1a.ycp";

    import "ProductFeatures";

    ProductFeatures::SetBooleanFeature("partitioning", "try_separate_home", true);
    ProductFeatures::SetBooleanFeature("partitioning", "proposal_lvm", true);
    ProductFeatures::SetBooleanFeature("partitioning", "vm_keep_unpartitioned_region", true);
    ProductFeatures::SetStringFeature("partitioning", "vm_desired_size", "30 GB");
    ProductFeatures::SetStringFeature("partitioning", "root_base_size", "20 GB");

    include "./lib/helper1b.ycp";
}
