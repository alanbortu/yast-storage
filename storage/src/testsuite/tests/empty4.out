Dump	Proposal:
Dump	Create boot volume /dev/sda1 (156.88 MB) with ext4
Dump	Create volume /dev/sda2 (29.98 GB)
Dump	Create volume group system (29.98 GB) from /dev/sda2 [destructive]
Dump	Create logical volume /dev/system/home (19.48 GB) for /home with ext4
Dump	Create logical volume /dev/system/root (10.00 GB) for / with ext4
Dump	Create swap logical volume /dev/system/swap (512.00 MB)
